addon/components/s-horizontal-menu-item.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+
```

Comment:
generate this file using ember g component
This component will not work as it doesnt have boilerplate code required for addon. like importing layout